src/components/link/index.stories.mdx

```diff
@@ -0,0 +1,55 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { ArrowBackIcon, ExternalLinkIcon } from '@chakra-ui/icons';
+
+import Link from './index.tsx';
+
+<Meta title="Components/Link" component={Link} />
+
+# Link
+
+## Overview
+
+<Canvas>
+  <Story name="Overview">
+    <Link href="https://autoscout24.ch">{'AutoScout24 Homepage'}</Link>
+  </Story>
+</Canvas>
+
+<Canvas>
+  <Story name="Link with Left icon">
+    <Link leftIcon={ArrowBackIcon}>Back</Link>
+  </Story>
+</Canvas>
+
+<Canvas>
+  <Story name="Link with Right icon">
+    <Link
+      rightIcon={ExternalLinkIcon}
+      href="https://google.com"
+      target="_blank"
+    >
+      {'Go to External Link'}
```

Comment:
Use JSX syntax instead of plain text to avoid `p` autogenerated from StoryBook (will make some line breaks)